.gitignore

```diff
@@ -25,3 +25,4 @@ doc/notebooks/Canary_Example_Train_Predict/annotated_prep_llb11_example.csv
 src/configs/Canaries/llb16/llb16_eval_080320.toml
 src/configs/Canaries/llb16/llb16_evade_080320.toml
 src/scripts/Canaries/eval.sh
+.vscode/launch.json
```

Comment:
please don't silently add change to .gitignore in a commit like this

if you need to ignore .vscode configs then just add it under the 'IDE' section with '.idea', but please do it in a separate commit where you make it explicit in the message, i.e. "add .vscode to IDE section of .gitignore"